spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/TokenStoreProperties.java

```diff
@@ -0,0 +1,41 @@
+package org.axonframework.springboot;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+import java.time.Duration;
+
+/**
+ * Properties describing the settings for the default Token Store.
+ *
+ * @author Gerard Klijs
+ * @since 4.8.0
+ */
+@ConfigurationProperties("axon.tokenstore")
```

Comment:
Wouldn't it be more reasonable to place these properties under `axon.eventhandling.tokenstore`?

Reply:
fixed